run.sh

```diff
@@ -57,6 +59,16 @@ run_and_configure_emulator() {
   export SPANNER_EMULATOR_HOST="localhost:9010"
 }
 
+generate_py_proto_srcs() {
+  local python_source=server/src/main/python
+  local proto_source=server/src/main/proto
+  python -m grpc_tools.protoc -I $proto_source \
+    --python_out=$python_source --grpc_python_out=$python_source \
+    service.proto
+  protoc --proto_path=$proto_source --python_out=$python_source \
```

Comment:
Where does `protoc` come from? I pip installed all the requirements but I do not have it in my linux environment.